src/configurationProvider.ts

```diff
@@ -202,6 +211,22 @@ export class JavaDebugConfigurationProvider implements vscode.DebugConfiguration
         }
     }
 
+    /**
+     * Converts an array of arguments to a string as the args and vmArgs.
+     */
+    private concatArgs(args: any[]): string {
+        return _.join(_.map(args, (arg: any): string  => {
+            const str = String(arg);
+            // if it has quotes or spaces, use double quotes to wrap it
+            if (/['"\s]/.test(str)) {
```

Comment:
What if the user already quote the arguments by himself/herself?

Reply:
I will quote it twice and use may notice this soon and remove his quotes.